### DISCUSSION TITLE
using in ASYNCIFY_ADD

### DISCUSSION THREAD

**Discussion by aharondavid** [emscripten-core/emscripten#17177] [Q&A]
Hi,
Is any one has an example of how to use in ASYNCIFY_ADD or ASYNCIFY_ONLY  ?

**Comment by kripken** [comment]
Both are used in the test suite, so you can find concrete working code for them there. Searching for those options (full text search, like `grep -r`) on `tests/` will find the places.

**Reply by aharondavid** [reply]
For reduced the build time with ASYNCIFY, I had used in ASYNCIFY_ONLY with only one function, but, unfortunately,  it didn’t reduced the build time.
what am i missing?

**Reply by kripken** [reply]
`ASYNCIFY_ONLY` might not affect build times much. Just running Asyncify, no matter with what options, causes the large overhead atm.

But you can try `-sASYNCIFY_IGNORE_INDIRECT` which will further reduce the functions that are modified. But I still wouldn't expect too much.
